scripts/dockerfiles/Dockerfile.metrics

```diff
@@ -6,17 +6,13 @@ WORKDIR /go/src/github.com/aws/amazon-vpc-cni-k8s
 ENV GO111MODULE=on
 ENV GOPROXY=direct
 
-# Copy modules in before the rest of the source to only expire cache on module
-# changes:
+# Copy modules in before the rest of the source to only expire cache on module changes:
 COPY go.mod go.sum ./
-RUN go mod download
```

Comment:
As discussed, let's hold off on these changes for now. What we've currently helps w.r.t caching the dep. modules. Let's discuss offline.

Reply:
sure, removed in latest update